Client/Frontend/Browser/EnhancedTrackingProtection/EnhancedTrackingProtectionVM.swift

```diff
@@ -31,13 +31,12 @@ class EnhancedTrackingProtectionMenuVM {
     }
 
     var connectionStatusImage: UIImage {
-        let insecureImageString = "lock_blocked"
-        let image = connectionSecure ? UIImage(named: "secureLock")! : UIImage(imageLiteralResourceName: insecureImageString)
-        return image
+        let connectionStatus: WebsiteConnectionTypeStatus = connectionSecure ? .secure : .unsecure
```

Comment:
Also wondering if we should add the `//Ecosia: ...` comment when changing an FF file, not sure if that is relevant for every change and actually helps when merging though 🤔

Reply:
I can add yeah 👍